package.json

```diff
@@ -1,51 +1,43 @@
 {
-  "name": "gosling-react-template",
+  "name": "gosling-react-vite-example",
```

Comment:
Again, I think that it's fine to keep as `gosling-react-template`. We are just migrating our recommended build tool (i.e., we didn't call it gosling-react-webpack-example).